src/main/java/com/github/lmiljak/dowertefence/OnCommandGameCreator.java

```diff
@@ -0,0 +1,276 @@
+
+		// Check if the sender wants to play single player, if so then just
+		// create the game with him in it
+		if (invitedPlayers.length == 0) {
+			participants.add(sender);
+			notifyListeners(new StandardGame(participants));
+			return true;
+		}
+
+		participants = invitePlayers(sender, invitedPlayers);
+		if (participants == null) {
+			// The invites have been cancelled and the sender got notified.
+			return true;
+		}
+
+		for (String arg : invitedPlayers) {
+			Player invitedPlayer = Bukkit.getPlayer(arg);
+
+			if (invitedPlayer == null) {
+				sender.sendMessage("One or more of the specified players do not exist or are not online.");
+				return true;
+			}
+
+			participants.add(invitedPlayer);
+		}
+
+		// Creating a new potential game.
+		potentialGames.put(sender, new PotentialGame(participants));
+		sender.sendMessage("Waiting for all invited players to accept.");
+		return true;
+	}
+
+	/**
+	 * Invites an array of players by sending them a message. If one or more of
+	 * the specified players do not exist or are not online, the invites will be
+	 * cancelled and the inviter will be notified.
+	 * 
+	 * @param inviter
+	 *            The player that invited the other players.
+	 * @param players
+	 *            The names of the players that should get invited.
+	 * @return The Set of players that got invited. Null if the invites got
+	 *         cancelled.
+	 */
+	private HashSet<Player> invitePlayers(Player inviter, String[] players) {
+		HashSet<Player> result = new HashSet<>();
+		for (String player : players) {
+			Player invitedPlayer = Bukkit.getPlayer(player);
+			if (invitedPlayer == null) {
+				inviter.sendMessage("One or more of the specified players do not exist or are not online.");
+				return null;
+			}
+			if (invitedPlayer != inviter) {
+				result.add(invitedPlayer);
+			}
+		}
+
+		for (Player player : result) {
+			player.sendMessage(
+					inviter.getName() + " has invited you to join a game of Dower Tefence. Type \"/dt accept "
+							+ inviter.getName() + "\" to accept the invite.S");
+		}
+
+		return result;
+	}
+
+	/**
+	 * Used to process the accept command. If a valid PotentialGame has been
+	 * found for this player, he will be marked as accepted. If all players in
+	 * that PotentialGame have accepted, the game is created.
+	 * 
+	 * @param sender
+	 *            The player that sent the command.
+	 * @param hostingPlayer
+	 *            The player that invited the sender (the arguments of the
+	 *            command).
+	 * @return True if and only if the command was executed successfully.
+	 */
+	private boolean processAcceptCommand(Player sender, String[] hostingPlayer) {
+		if (hostingPlayer.length != 1) {
+			// The sender failed to follow the usage of this command.
+			return false;
+		}
+
+		Player host = Bukkit.getPlayer(hostingPlayer[0]);
+		if (host == null) {
+			sender.sendMessage("The specified players does not exist or is not online.");
```

Comment:
This sentence switches from plural to singular mid sentence and as a result doesn't make sense.